views/index.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+  <meta charset="UTF=8">
+  <title>My Suite App</title>
+</head>
+<body>
+  <h1>Hi!</h1>
+  <%= @phrases.sample %> This runs Ruby code
```

Comment:
This line could also be removed. Also think about how you might combine the stuff that you are returning with this erb template.